save.py

```diff
@@ -195,6 +196,24 @@ def rare_chance_as_str(self) -> tuple[str, str, str]:
     def upcoming_boss(self) -> str:
         return self._data["boss"]
 
+    @property
+    def bottled_cards(self) -> str:
+        cards = []
+        if self._data["bottled_flame"]:
+            cards.append(self.__get_card_string(self._data["bottled_flame"], self._data["bottled_flame_upgrade"]))
+        if self._data["bottled_lightning"]:
+            cards.append(self.__get_card_string(self._data["bottled_lightning"], self._data["bottled_lightning_upgrade"]))
+        if self._data["bottled_tornado"]:
+            cards.append(self.__get_card_string(self._data["bottled_tornado"], self._data["bottled_tornado_upgrade"]))
+        return cards    
+
+    def __get_card_string(self, card: str, upgrades: int) -> str:
+        if upgrades == 1:
+            card += "+"
+        elif upgrades > 1:
+            card += "+" + str(upgrades)
```

Comment:
`get_card` automatically converts `card+1` into `card+`. The following would probably work better:
```
    if upgrades:
        card = f"{card}+{upgrades}"
    return get_card(card)
```